stbt_rig.py

```diff
@@ -329,6 +356,25 @@ def cmd_run(args, node):
             value = os.environ.get(v.replace(".", "_"))
             if value:
                 tags[v] = value
+    elif args.mode == "gitlab":
+        # Record gitlab CI variables as Stb-tester tags.
+        # Gitlab CI changed the variables names in version 9.0 (released
+        # 2017-03-22).  We support both sets of names, but always record the
+        # new name.
+        # See https://docs.gitlab.com/ee/ci/variables/
+        #                    8.x name      9.0+ name
+        for k, old_key in [('CI_JOB_ID', 'CI_BUILD_ID'),
```

Comment:
I think the comment is the wrong way round. The current [gitlab docs](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html#predefined-variables-reference) list CI_JOB_* but not CI_BUILD_*.

Reply:
Ah yes, the code is correct, just the comment is wrong.